projects/rna/README.md

```diff
@@ -0,0 +1 @@
+TODO: Create the readme file.
```

Comment:
This probably doesn't belong here either. If we did have a file here, it would just be a pointer to [docs/monorepo.md](https://github.com/Automattic/jetpack/blob/master/docs/monorepo.md) that explains the structure of the whole monorepo.